graphics/cursorman.h

```diff
@@ -31,19 +31,28 @@
 
 namespace Graphics {
 
+/**
+ * @defgroup graphics_cursorman Cursor manager
+ * @ingroup graphics
+ *
+ * @brief  The CursorManager class for managing the behavior of the mouse cursor.
+ *
+ * @{
+ */
+
 class CursorManager : public Common::Singleton<CursorManager> {
 public:
-	/** Query whether the mouse cursor is visible. */
+	/** Return whether the mouse cursor is visible. */
 	bool isVisible();
 
 	/**
 	 * Show or hide the mouse cursor.
 	 *
 	 * This function does not call OSystem::updateScreen, when visible is true.
-	 * This fact might result in a non visible mouse cursor if the caller does
-	 * not call OSystem::updateScreen itself after a showMouse(true) call.
+	 * This might result in a non-visible mouse cursor if the caller does
+	 * not call OSystem::updateScreen after a showMouse(true) call.
```

Comment:
Maybe rewrite that sentence to improve the flow.
I would for example suggest something like: `You need to call OSystem::updateScreen after a call to showMouse(true) to ensure that the mouse cursor becomes visible.`